### DISCUSSION TITLE
GitHub Pages deployment

### DISCUSSION THREAD

**Discussion by f-laurens** [nabaztag2018/pynab#261] [General]
The GitHub Pages website for the repository is current broken.
It might be nice to reactivate it (from https://github.com/nabaztag2018/pynab/settings/pages).

**Comment by mehdichaouch** [comment]
This is usefull when a repository host a static website like [here](https://github.com/mehdichaouch/screen-cheat-sheet), but as I know there is no static website in pynab? right @omevel?

**Comment by f-laurens** [comment]
> This is useful when a repository hosts a static website, but as far as I know there is no static website in pynab?

Indeed.
But since deploying a `docs` subdirectory is supported by GitHub Pages, its a simple matter of creating such a subdirectory in pynab.
Then at least put `README.md` there (and turn the original `README.md` into a symlink to `./docs/README.md`).

**Reply by f-laurens** [reply]
Even simpler than that (no `docs` subdirectory shuffle needed):
just create a file `index.md` symlinked to `README.md`, and deploy the` / (root)` directory in GitHub pages...

**Reply by mehdichaouch** [reply]
If the goal of the `index.md` or `index.html` is to host the same things as `README.md`, imo is useless.
You can do an exemple of what you want to do in your fork repository this will be maybe more understandable.

**Reply by f-laurens** [reply]
> If the goal of the `index.md` or `index.html` is to host the same things as `README.md`, imo is useless.

That's a point of view: the goal is just to have a nice looking visibilty on github.io **with minimal (zero) effort**.
(if we did only 'useful' things, I guess we would not be playing with rabbits...)
> You can do an exemple of what you want to do in your fork repository this will be maybe more understandable.

Here it is : https://f-laurens.github.io/pynab/
PR #264

**Reply by mehdichaouch** [reply]
> the goal is just to have a nice looking

That your point of view !
IMO having the same content is just redundance and not necessary.

**Reply by f-laurens** [reply]
> That your point of view !

Indeed. We obviously have incompatible point of views on this matter: I do not care about redundancy, if it comes at zero cost; I don't care about necessity neither.
Any further discussion on this seems pointless.

**Comment by f-laurens** [comment]
Looks like this discussion may be closed.
Now it's up to the powers in charge to activate GitHub Pages deployment... or not...
